Formula/fstar.rb

```diff
@@ -8,13 +8,14 @@ class Fstar < Formula
 
   bottle do
     cellar :any_skip_relocation
-    revision 1
+    revision 2
```

Comment:
Sorry, please put this back to `revision 1` and add a `revision 1` line below `head` above, thanks.